common/rfb/SSecurityTLS.h

```diff
@@ -35,6 +35,7 @@
 #include <rdr/InStream.h>
 #include <rdr/OutStream.h>
 #include <gnutls/gnutls.h>
+#include <gnutls/x509.h>
```

Comment:
Please avoid putting more includes in headers than necessary. This can be added to `SSecurityTLS.cxx` instead.

Otherwise it looks good. Nice work.